portal/src/main/webapp/js/src/plots-tab/view/components/clinSpec.js

```diff
@@ -3,9 +3,7 @@ var clinSpec = (function() {
     function appendAttrList(axis) {
         $("#" + ids.sidebar[axis].spec_div).append("<br><h5>Clinical Attribute</h5>");
         $("#" + ids.sidebar[axis].spec_div).append("<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<select class='chosen' style='max-width: 250px;' id='" + ids.sidebar[axis].clin_attr + "'>");
-        //TODO: this is a temp solution (duplications in API results)
-        var _uniqClinMeta = _.uniq(metaData.getClinAttrsMeta(), "id");
-        $.each(_uniqClinMeta, function(index, obj) {
+        $.each(metaData.getClinAttrsMeta(), function(index, obj) {
```

Comment:
@oplantalech is this change really needed?

Reply:
Well, this is filtering all clinical variables by their id. Theoretically, with the new fix, they shouldn't be duplicates, otherwise this would mean that we have two clinical variables in the same study with the same attribute name (id; not to confuse with display name).